packages/cli/src/commands/destroy/page/page.js

```diff
@@ -33,6 +33,8 @@ export const tasks = ({ name, path }) =>
           const f = pageFiles({
             name,
             path: p,
+            stories: true,
```

Comment:
Might Redwood also need to do the same for Layouts?

There are destroy commands for:

* cells
* components
* pages
* layouts

Reply:
I am checking. Layouts already works because uses the same helper `packages/cli/src/commands/destroy/helpers.js` `createYargsForComponentDestroy`, I'll add a test, is using a similar function to get the layoutFiles:
```bash
pablo@galagopro ~/p/testredwoodapp (master)> yarn rw g layout Blog; yarn rw d layout Blog
yarn run v1.22.5
$ /home/pablo/projects/testredwoodapp/node_modules/.bin/rw g layout Blog
  ✔ Generating layout files...
    ✔ Successfully wrote file `./web/src/layouts/BlogLayout/BlogLayout.test.js`
    ✔ Successfully wrote file `./web/src/layouts/BlogLayout/BlogLayout.stories.js`
    ✔ Successfully wrote file `./web/src/layouts/BlogLayout/BlogLayout.js`
Done in 4.45s.
yarn run v1.22.5
$ /home/pablo/projects/testredwoodapp/node_modules/.bin/rw d layout Blog
  ✔ Destroying layout files...
    ✔ Destroying `./web/src/layouts/BlogLayout/BlogLayout.test`...
    ✔ Destroying `./web/src/layouts/BlogLayout/BlogLayout.stories`...
    ✔ Destroying `./web/src/layouts/BlogLayout/BlogLayout`...
    ✔ Cleaning up empty directories...
      ✔ Removing empty `./web/src/layouts/BlogLayout`...
Done in 4.50s.
```

Cells also work because they call the same helper:
```
pablo@galagopro ~/p/testredwoodapp (master)> yarn rw g cell BlogPosts
yarn run v1.22.5
$ /home/pablo/projects/testredwoodapp/node_modules/.bin/rw g cell BlogPosts
  ✔ Generating cell files...
    ✔ Successfully wrote file `./web/src/components/BlogPostsCell/BlogPostsCell.mock.js`
    ✔ Successfully wrote file `./web/src/components/BlogPostsCell/BlogPostsCell.test.js`
    ✔ Successfully wrote file `./web/src/components/BlogPostsCell/BlogPostsCell.stories.js`
    ✔ Successfully wrote file `./web/src/components/BlogPostsCell/BlogPostsCell.js`
Done in 4.06s.
pablo@galagopro ~/p/testredwoodapp (master)> yarn rw d cell BlogPosts
yarn run v1.22.5
$ /home/pablo/projects/testredwoodapp/node_modules/.bin/rw d cell BlogPosts
  ✔ Destroying cell files...
    ✔ Destroying `./web/src/components/BlogPostsCell/BlogPostsCell.mock`...
    ✔ Destroying `./web/src/components/BlogPostsCell/BlogPostsCell.test`...
    ✔ Destroying `./web/src/components/BlogPostsCell/BlogPostsCell.stories`...
    ✔ Destroying `./web/src/components/BlogPostsCell/BlogPostsCell`...
    ✔ Cleaning up empty directories...
      ✔ Removing empty `./web/src/components/BlogPostsCell`...
Done in 4.08s.
```